pkgs/pkgs-lib/formats.nix

```diff
@@ -34,6 +34,8 @@ rec {
   inherit (import ./formats/java-properties/default.nix { inherit lib pkgs; })
     javaProperties;
 
+  libconfig = (import ./formats/libconfig/default.nix { inherit lib pkgs; }).format;
```

Comment:
I believe there's a significant difference between these two. `json` only uses `callPackage` inside its `generator`, not for importing the entire format function. But since it ends up as a function, I guess it doesn't matter. I'll go ahead an put it on the generator as well